nf_core/__main__.py

```diff
@@ -732,8 +732,16 @@ def create_test_yml(ctx, tool, run_tests, output, force, no_prompts):
 @click.option("-w", "--fail-warned", is_flag=True, help="Convert warn tests to failures")
 @click.option("--local", is_flag=True, help="Run additional lint tests for local modules")
 @click.option("--passed", is_flag=True, help="Show passed tests")
+@click.option(
```

Comment:
This is for `nf-core modules lint` but maybe you also want it for `nf-core lint`?

Reply:
Added to pipeline linting. Called it `sort_by` for consistency across commands but did consider if it should be `module_sort_by` in pipeline lint

Reply:
I personally like `sort_by` more, I find `module_sort_by` a bit more confusing. Should we merge? are you happy with it @awgymer ?

Reply:
Ok that's good then. Happy to merge if this gets an approving review

Reply:
sorry, I thought it was already approved 😅